lib/Models/Definition/updateModelFromJson.ts

```diff
@@ -25,7 +25,8 @@ export default function updateModelFromJson(
         propertyName === "id" ||
         propertyName === "type" ||
         propertyName === "localId" ||
-        propertyName === "shareKeys"
+        propertyName === "shareKeys" ||
+        propertyName === "aspects"
```

Comment:
I don't `updateModelFromJson` should have `Magda` specific things like this - it should only handle Terria model JSON.

Removing this would follow from my previous comment suggestion

Reply:
Removed.